frontend/src/main/components/Sections/SectionsTable.js

```diff
@@ -14,7 +14,7 @@ function getFirstVal(values) {
   return values[0];
 }
 
-export default function SectionsTable({ sections }) {
+export default function SectionsTable(props) {
```

Comment:
```suggestion
export default function SectionsTable({sections, canExpand=true}) {
```

If, instead, we make this a prop that describes the functionality and has a default value of the old behavior, then it will be backwards compatible with existing components, and easy to use the alternative behavior in future components.